cmake/projects/ZMQPP/hunter.cmake

```diff
@@ -27,6 +27,17 @@ hunter_add_version(
   fdc3a0d5a4e907c79709ec2fc3fcf4105ff1b91a
   )
 
+hunter_add_version(
+  PACKAGE_NAME
+  ZMQPP
+  VERSION
+  "4.2.0"
```

Comment:
Please keep Git tag and `VERSION` in a consistent state:
- https://docs.hunter.sh/en/latest/creating-new/create/cmake.html#consistency